bucket/cuda80.json

```diff
@@ -0,0 +1,32 @@
+{
+    "version": "8.0.44",
+    "description": "A parallel computing platform and programming model invented by NVIDIA",
+    "homepage": "https://developer.nvidia.com/cuda-toolkit",
+    "license": {
+        "identifier": "Freeware",
+        "url": "https://docs.nvidia.com/cuda/eula/index.html"
+    },
+    "architecture": {
+        "64bit": {
+            "url": "https://developer.nvidia.com/compute/cuda/8.0/prod/local_installers/cuda_8.0.44_win10-exe#/dl.7z",
+            "hash": "md5:a7088813a5e40e51296df54e06496c54"
```

Comment:
If no `checkver`, why you don't use SHA-256?

Reply:
Because there is an available checksum at https://developer.download.nvidia.com/compute/cuda/8.0/Prod/docs/sidebar/md5sum.txt

Or, should I just download the file and use **SHA-256**?

Reply:
Yes, you should download it and hashcheck with SHA-256, since MD5 has less security.